emuu-client/src/components/upload/UploadButton.js

```diff
@@ -191,63 +470,328 @@ function FileUpload() {
+                          <MenuItem value="Survival">Survival</MenuItem>
+                          <MenuItem value="Other">Other</MenuItem>
+                        </Select>
+                        <FormHelperText>Required</FormHelperText>
+                      </FormControl>
+                      <label htmlFor="upload-photo">
+                        <input
+                          style={{ display: "none" }}
+                          id="upload-thumbnail"
+                          name="upload-thumbnail"
+                          type="file"
+                          onChange={handleThumbnailChange}
+                          accept="image/jpeg"
+                        />
+                        <Button
+                          className={classes.btnClass}
+                          type="submit"
+                          variant="contained"
+                          component="span"
+                          onClick={() =>
+                            document.querySelector("#upload-thumbnail").click()
+                          }
+                        >
+                          UPLOAD THUMBNAIL
+                        </Button>
+
+                        {thumbnail && (
+                          <img
+                            src={thumbnailPreview}
+                            alt="thumbnail-alt"
+                            height="80"
+                            style={{ marginLeft: "5px" }}
+                          />
+                        )}
+                      </label>
+                    </Box>
+                  </div>
+                  <div style={{ marginLeft: "30px", marginTop: "55px" }}>
+                    <Video
+                      id="1"
+                      height="200"
+                      src={videoPreview}
+                      title={file?.name?.split(".mp4")[0]}
+                    />
+                  </div>
+                </div>
+                <Button
+                  className={classes.submitBtn}
+                  type="submit"
+                  variant="contained"
+                  component="span"
+                  onClick={onFormSubmit}
+                >
+                  SUBMIT
+                </Button>
+              </div>
+            </Dialog>
+          </div>
         </div>
-      </form>
-      <h2>Please Choose a Video</h2>
-      <input type="file" onChange={handleChange} accept="video/mp4" />
-      <br />
-      <br />
-
-      <h2>Please Choose a Thumbnail</h2>
-      <input type="file" onChange={handleThumbnail} accept="image/jpeg" />
-
-      <p>
-        {" "}
-        <LinearProgressWithLabel value={percent} />{" "}
-      </p>
-      <button
-        onClick={() => handleUpload()}
-        type="submit"
-        className="btn btn-primary"
-      >
-        Upload
-      </button>
-      {uploadStatus}
-    </div>
+      )}
+    </AppContext.Consumer>
   );
-}
+};
```

Comment:
Good job on getting upload page done to reflect a clean UI and be similar to YouTube.